src/js/vue.js

```diff
@@ -1,9 +1,11 @@
 import route from './index.js';
 
 export const ZiggyVue = {
-    install: (v, options) => v.mixin({
-        methods: {
-            route: (name, params, absolute, config = options) => route(name, params, absolute, config),
-        },
-    }),
+    install: (v, options) => {
+        const version = parseInt(v.version.split('.')[0]);
+        const _route = (name, params, absolute, config = options) => route(name, params, absolute, config);
+
+        v.mixin && version === 2 && (v.mixin({ methods: { route: _route } }));
```

Comment:
I'm on board with this in principle but it breaks backwards compatibility for all existing Vue 3 apps using Ziggy. We have to keep the mixin for now.

Reply:
I'd have to check again, but if a user needs `route()` in their `<script setup>` and uses `inject()` I'm pretty sure it will conflict with the `mixin route()` loaded automatically in the `<template>`

Reply:
Interesting, "conflict" how? Would it error or would one just override the other, which shouldn't matter? I'll try it in the sandbox again and see what happens.

Reply:
Ugh, sorry I don't remember but I pulled it last night cause it was breaking...but I may have screwed something up. If it works, we should keep it then! Thanks for helping.

If it works, I will update the PR again today adding back the mixin.

Reply:
Looks to me like it works fine?

https://stackblitz.com/edit/vitejs-vite-pdjx4q?file=src%2FSetup.vue&terminal=dev

Reply:
Yupp, looks great. I will make the syntax tweaks and put back the mixin and push them up shortly.